### ISSUE TITLE
Message component - basic functionality

### ISSUE THREAD

**Issue by szuperaz** [GetStream/stream-chat-angular#16]
Inputs:
- message to be displayed

Behavior:
- Display the avatar (blocked by avatar #15)
- Display message "bubble"
- Display delivery/sent date (`MessageTimestamp` component in React SDK)
- Display message status (sent, error etc.) (`MessageStatus` component in React SDK)

Out of scope
- Message group styles (e.g. only show the avatar and date for the last message in a group)
- Retry sending unsent messages on click
- Delete message
- Edit message
- Quoted message
- Emoji
- HTML messages
- Markdown parsing
- Translation
